src/server/api/campaign.js

```diff
@@ -76,6 +81,13 @@ export const resolvers = {
         .eqJoin('id', r.table('message'), { index: 'assignment_id' })
         .filter({ is_from_contact: true })
         .count()
+    ),
+    optOutsCount: async (campaign) => (
+      r.table('campaign_contact')
```

Comment:
We're trying to stick with knex for newer queries, so this would be `r.knex('campaign_contact').where({'is_opted_out': true, 'campaign_id': campaign.id}).count()`